mm2src/mm2_tests.rs

```diff
@@ -7132,11 +7132,32 @@ fn test_sign_message_utxo() {
         signature,
         "HzetbqVj9gnUOznon9bvE61qRlmjH5R+rNgkxu8uyce3UBbOu+2aGh7r/GGSVFGZjRnaYC60hdwtdirTKLb7bE4="
     );
+
+    let rc = block_on(mm_bob.rpc(&json! ({
+    "userpass": mm_bob.userpass,
+    "method":"verify_message",
+    "mmrpc":"2.0",
+    "id": 0,
+    "params":{
+      "coin":"RICK",
+      "message":"test",
+      "signature": "HzetbqVj9gnUOznon9bvE61qRlmjH5R+rNgkxu8uyce3UBbOu+2aGh7r/GGSVFGZjRnaYC60hdwtdirTKLb7bE4=",
+      "address":"R9o9xTocqr6CeEDGDH6mEYpwLoMz6jNjMW"
+
+    }
+    })))
+    .unwrap();
+
+    assert!(rc.0.is_success(), "!verify_message: {}", rc.1);
+
+    let response: Json = json::from_str(&rc.1).unwrap();
+    let is_valid = &response["result"]["is_valid"];
+    assert_eq!(is_valid, true);
```

Comment:
`assert!(is_valid)`

Reply:
`is_valid` is not a bool here, it's a json value.

Reply:
Oh, I see. However, it will be still actual with a specific response type as mentioned here https://github.com/KomodoPlatform/atomicDEX-API/pull/1256#discussion_r863562290 :slightly_smiling_face:

Reply:
Right, early comment by me...